src/Azs.Backup.Admin/test/Get-AzsBackupConfiguration.Tests.ps1

```diff
@@ -46,7 +57,7 @@ Describe 'Get-AzsBackupConfiguration' {
         $backupLocations = Get-AzsBackupConfiguration -Top 10
         $backupLocations  | Should Not Be $null
         foreach ($backupLocation in $backupLocations) {
-            $result = $backupLocation | Get-AzsBackupConfiguration
+            $result = Get-AzsBackupConfiguration -InputObject $backupLocation
```

Comment:
Can you please add a comment why the pipelining is not preferred here and possibly the link to the issue in the comments?

Reply:
You get this error if you don't use parameter:

    [-] TestRestoreBackupViaIdentityExpanded 159ms
      InvalidOperationException: The context is invalid. Please login using Connect-AzAccount.
      at Get-AzsBackup<Process>, D:\a\1\s\src\Azs.Backup.Admin\internal\ProxyCmdletDefinitions.ps1: line 180
      at Get-AzsBackup<Process>, D:\a\1\s\src\Azs.Backup.Admin\custom\Get-AzsBackup.ps1: line 137
      at Restore-AzsBackup<Process>, D:\a\1\s\src\Azs.Backup.Admin\custom\Restore-AzsBackup.ps1: line 181
      at Restore-AzsBackup<Process>, D:\a\1\s\src\Azs.Backup.Admin\exports\Restore-AzsBackup.ps1: line 226
      at <ScriptBlock>, D:\a\1\s\src\Azs.Backup.Admin\test\Restore-AzsBackup.Tests.ps1: line 61

Reply:
What Bala meant was, to add a comment inside the test file before line like something like this:
`## TODO: Workaround for issue with pipeline: https://github.com/Azure/autorest.powershell/issues/606`